<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliderForm</class>
 <widget class="QWidget" name="SliderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboBoxCountries"/>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxWeekly"/>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSliderIntRange">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxInt"/>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxStruct"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
